extension/manifest.json

```diff
@@ -112,7 +113,8 @@
 				"scripts/features/hide-icons/ttHideIcons.entry.js",
 				"scripts/features/block-zalgo/ttBlockZalgo.entry.js",
 				"scripts/features/achievements/achievements.js",
-				"scripts/features/achievements/ttAchievements.entry.js"
+				"scripts/features/achievements/ttAchievements.entry.js",
+				"scripts/features/user-alias/ttUserAlias.js"

```

Comment:
Either the script is in the wrong place (document_start) or it should be ending with `.entry.js`.